src/routes/v2/authenticatedSites/collectionPages.ts

```diff
@@ -54,20 +66,37 @@ class CollectionPagesRouter {
           collectionName,
           content: pageBody,
           frontMatter,
+          shouldIgnoreCheck: false,
```

Comment:
@alexanderleegs hey, tagging you here to just double confirm this should be the correct value